torch/_inductor/ir.py

```diff
@@ -3882,7 +3882,11 @@ def generate_output(output, indices):
                 assert output is None, "FallbackKernel output type is not supported"
                 return None
 
-        packed.outputs = generate_output(example_output, [])
+        outputs = generate_output(example_output, [])
+        if isinstance(outputs, (list, tuple)):
+            packed.outputs = outputs
+        else:
+            packed.outputs = [outputs]
```

Comment:
Wondering if there is some other code that relies of the `outputs` occasionally not being `(list, tuple)`? I guess, we'll see it in the CI.